shared/yarn.lock

```diff
@@ -2022,6 +2022,86 @@
+  integrity sha512-3oHpAxh+kWwGUm9Rmp4mc2J4uHCigu72wC0NWyEF0SvbyctKQgx530CTYvaqUVQ94QB3HcBR9v0enRMWUqXtVQ==
+  dependencies:
+    "@typescript-eslint/scope-manager" "5.11.1-alpha.10+27946318"
+    "@typescript-eslint/types" "5.11.1-alpha.10+27946318"
+    "@typescript-eslint/typescript-estree" "5.11.1-alpha.10+27946318"
+    debug "^4.3.2"
+
+"@typescript-eslint/scope-manager@5.11.1-alpha.10+27946318":
+  version "5.11.1-alpha.10"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/scope-manager/-/scope-manager-5.11.1-alpha.10.tgz#50c8d02b59d8e75004d867ecffd4d6bf055e6bac"
+  integrity sha512-amF+cQroczrJ9M7omlagb2rbKTmSqwXdPPAe5rWMstD8QxhsO8lXqU6/crljDajoikbOvH8kNCIz8+bnnvaQDA==
+  dependencies:
+    "@typescript-eslint/types" "5.11.1-alpha.10+27946318"
+    "@typescript-eslint/visitor-keys" "5.11.1-alpha.10+27946318"
+
+"@typescript-eslint/type-utils@5.11.1-alpha.10+27946318":
+  version "5.11.1-alpha.10"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/type-utils/-/type-utils-5.11.1-alpha.10.tgz#b22a36479f070fa096af30201e347095e6f8a004"
+  integrity sha512-mXs62DJvSXmAeZpTQH3y6egZKDuEBr/D6spE9r3669PgpcJtVgLYYoblqRAm8baD2Yh6qK8IbgPutH24D681TA==
+  dependencies:
+    "@typescript-eslint/utils" "5.11.1-alpha.10+27946318"
+    debug "^4.3.2"
+    tsutils "^3.21.0"
+
+"@typescript-eslint/types@5.11.1-alpha.10+27946318":
+  version "5.11.1-alpha.10"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/types/-/types-5.11.1-alpha.10.tgz#e1a6ea443368e1b62ff54760663f88e8d233f49a"
+  integrity sha512-EWDVBuDF5L8VgF7olrNaGqseIFBpyMmX8qehasV1hXDLUljVwmT46efz0Xt4D68w3iHcevmE8fcFUHg/QQaRWA==
+
+"@typescript-eslint/typescript-estree@5.11.1-alpha.10+27946318":
+  version "5.11.1-alpha.10"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/typescript-estree/-/typescript-estree-5.11.1-alpha.10.tgz#d13983effa7838dc72258a91c6f81501f782fc0f"
+  integrity sha512-2B2v9imqAWFbJ/4ANDjvLfpfS1A5KSt6z57ntO3yUhPIg0/+HYgpxgodwupPiqHaGYJh6Cc5GWd5SVBirY+WJA==
+  dependencies:
+    "@typescript-eslint/types" "5.11.1-alpha.10+27946318"
+    "@typescript-eslint/visitor-keys" "5.11.1-alpha.10+27946318"
+    debug "^4.3.2"
+    globby "^11.0.4"
+    is-glob "^4.0.3"
+    semver "^7.3.5"
+    tsutils "^3.21.0"
+
+"@typescript-eslint/utils@5.11.1-alpha.10+27946318":
+  version "5.11.1-alpha.10"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/utils/-/utils-5.11.1-alpha.10.tgz#7d51eafff61280086149ace709bae26973353147"
+  integrity sha512-e5q7QSBq/Y/uGMMPXYMWSAzhjiMJMImdc6tXvReB8BCJgc17vHu+jC6fOYujMpYGTQHTQG6X8GcKh9g7rHAhnw==
+  dependencies:
+    "@types/json-schema" "^7.0.9"
+    "@typescript-eslint/scope-manager" "5.11.1-alpha.10+27946318"
+    "@typescript-eslint/types" "5.11.1-alpha.10+27946318"
+    "@typescript-eslint/typescript-estree" "5.11.1-alpha.10+27946318"
+    eslint-scope "^5.1.1"
+    eslint-utils "^3.0.0"
+
+"@typescript-eslint/visitor-keys@5.11.1-alpha.10+27946318":
```

Comment:
are any of these new dependencies worth looking at carefully? or are they all popular enough we shouldn't worry?